float32 left_pwm
float32 right_pwm
bool left_reverse
bool right_reverse